collectd/files/collectd.conf

```diff
@@ -1,16 +1,28 @@
-Hostname "{{ hostname }}"
-FQDNLookup true
+{%- from "collectd/map.jinja" import collectd_settings with context -%}
+
+#
+# DO NOT EDIT
+#
+# This file is managed by salt via {{ source }}
+# Modify the config that generates this file instead
+#
+
+Hostname "{{ salt['grains.get']('fqdn') }}"
+FQDNLookup {{ collectd_settings.FQDNLookup }}
 #BaseDir "/var/lib/collectd"
 #PluginDir "/usr/lib/collectd"
-TypesDB {% for type in types %} "{{ type }}" {% endfor %}
+
+{% for type in collectd_settings.TypesDB %}
+TypesDB "{{ type }}"
+{%- endfor %}
 #Interval 10
```

Comment:
This renders differently I think

```
TypesDB "type1" "type2" "type3"
```
vs
```
TypesDB "type1"
TypesDB "type2"
TypesDB "type3"
```
Not sure which one is exactly right.